python3.8/cookiecutter-aws-sam-efs-python/{{cookiecutter.project_name}}/README.md

```diff
@@ -57,13 +57,16 @@ To simplify troubleshooting, SAM CLI has a command called `sam logs`. `sam logs`
 
 You can find more information and examples about filtering Lambda function logs in the [SAM CLI Documentation](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-logging.html).
 
-## Unit tests
+## Tests
 
-Tests are defined in the `tests` folder in this project. Use PIP to install the [pytest](https://docs.pytest.org/en/latest/) and run unit tests.
+Tests are defined in the `tests` folder in this project. Use PIP to install the test dependencies and run tests.
 
 ```bash
-{{ cookiecutter.project_name }}$ pip install pytest pytest-mock --user
-{{ cookiecutter.project_name }}$ python -m pytest tests/ -v
+{{ cookiecutter.project_name }}$ pip install -r tests/requirements.txt --user
+# unit test
+{{ cookiecutter.project_name }}$ python -m pytest tests/unit -v
+# integration test, requiring deploying the stack first
+{{ cookiecutter.project_name }}$ AWS_SAM_STACK_NAME=<stack-name> python -m pytest tests/integration -v
```

Comment:
Can we please mention in comments that this should be the name of stack which we are testing.

Reply:
🆗